collectors/logsmanagement.plugin/plugin_logsmanagement.h

```diff
@@ -0,0 +1,59 @@
+/** @file plugin_logsmanagement.h
+ *
+ *  @author Dimitris Pantazis
+ */
+
+#ifndef PLUGIN_LOGSMANAGEMENT_H_
+#define PLUGIN_LOGSMANAGEMENT_H_
+
+#include "../../daemon/common.h"
+#include "../../logsmanagement/file_info.h"
+#include "../../logsmanagement/circular_buffer.h"
+
+struct Chart_meta;
+
+#include "plugin_logsmanagement_generic.h"
+#include "plugin_logsmanagement_web_log.h"
+#include "plugin_logsmanagement_systemd.h"
+#include "plugin_logsmanagement_docker_ev.h"
+
+/* NETDATA_CHART_PRIO for Stats_chart_data */
```

Comment:
These next `NETDATA_CHART_PRIO_*` has direct relationship with content inside [this](https://github.com/netdata/netdata/blob/master/collectors/all.h) file, I suggest you moved them for the file, instead to keep here. This will avoid future collectors to use them by mistake.

Reply:
@thiagoftsm Have a look at https://github.com/netdata/netdata/pull/13291/commits/60ad8f86f1116a8416abc96878c08c65db18bbc4 , I moved what (in my opinion) needs to be public to `collectors/all.h` and updated how priorities work to respect the order that the user has defined in `logsmanagement.conf`. They work in a more dynamic way now.

Reply:
I loved your solution, now new developers and contributors know they cannot use these values. :) 
I am marking this as resolved. :handshake: